pkg/agent/openflow/pipeline.go

```diff
@@ -2410,9 +2407,36 @@ func (f *featureService) serviceLBFlow(groupID binding.GroupIDType,
 	if f.enableAntreaPolicy {
 		flowBuilder = flowBuilder.Action().LoadToRegField(ServiceGroupIDField, uint32(groupID))
 	}
+	if nested {
+		// `nested` is true only when a Service is a Multi-cluster Service for now.
```

Comment:
I feel the comment belongs to where nested is calculated here, adding here still makes it specific to Multi-cluster while it doesn't have to.

Reply:
Done

Reply:
don't see any change